ipykernel/inprocess/client.py

```diff
@@ -172,6 +172,9 @@ def _dispatch_to_kernel(self, msg):
         idents, reply_msg = self.session.recv(stream, copy=False)
         self.shell_channel.call_handlers_later(reply_msg)
 
+    def flush(self):
+        # dummy method (see https://github.com/ipython/ipykernel/pull/405)
```

Comment:
Needs a docstring or `pass` to be valid syntax here.

```suggestion
        """no-op to comply with stream API"""
```